src/gt4py/next/program_processors/runners/dace_iterator/itir_to_sdfg.py

```diff
@@ -124,6 +125,17 @@ def add_storage(self, sdfg: dace.SDFG, name: str, type_: ts.TypeSpec, has_offset
             raise NotImplementedError()
         self.storage_types[name] = type_
 
+    def get_output_nodes(
+        self, closure: itir.StencilClosure, context: Context
+    ) -> dict[str, dace.nodes.AccessNode]:
+        if isinstance(closure.output, itir.SymRef):
+            name = str(closure.output.id)
+            return {name: context.state.add_access(name)}
+        else:
+            translator = PythonTaskletCodegen(self.offset_provider, context, self.node_types)
+            output_nodes = flatten_list(translator.visit(closure.output))
+            return {node.value.data: node.value for node in output_nodes}
```

Comment:
Can you extract common code from the two branches? Why not just `translator.visit` the output anyways, even if it's just a single one? Does `flatten_list` also handle single outputs? If so, you can maybe eliminate the `if` statement completely. I'm not saying you should at all costs, but if you don't have to change any of the rest of the code, you might as well.